commands/FBPrintCommands.py

```diff
@@ -299,3 +300,15 @@ def run(self, arguments, options):
       object = fb.evaluateObjectExpression(objectToMessage)
       printCommand = 'po [{} valueForKeyPath:@"{}"]'.format(object, keypath)
       lldb.debugger.HandleCommand(printCommand)
+
+
+class FBPrintApplicationDocumentsPath(fb.FBCommand):
+  def name(self):
+    return 'pdocspath'
+
+  def description(self):
+    return "Print application's 'Documents' directory path."
+
+  def run(self, arguments, options):
+    lldb.debugger.HandleCommand('po [NSSearchPathForDirectoriesInDomains(9, 1, YES) lastObject]')  # NSDocumentDirectory == 9 NSUserDomainMask == 1
```

Comment:
In other parts of Chisel, in addition to printing a value, we've copied it to the pasteboard. Would you mind adding that here?